spec/test_helper.rb

```diff
@@ -43,3 +47,20 @@ def printer.end(*) [] ;end
 def throttler
   Lhm::Throttler::Time.new(:stride => 100)
 end
+
+def init_test_db
+  db_config = YAML.load_file(File.expand_path(File.dirname(__FILE__)) + '/integration/database.yml')
+  conn = Mysql2::Client.new(
+    :host => '127.0.0.1',
+    :username => db_config['master']['user'],
+    :password => db_config['master']['password'],
+    :port => db_config['master']['port']
+  )
+
+  conn.query("DROP DATABASE IF EXISTS #{$db_name}")
+  conn.query("CREATE DATABASE #{$db_name}")
+end
+
+init_test_db
```

Comment:
From my understanding, `require` will only load a file and execute it if it has not been loaded yet. Meaning, this will only be loaded and executed once. At least this is my understanding of it, and there could be some edge cases when it comes to test environments where they could be scrapped after each test. [source](https://stackoverflow.com/questions/22757386/ruby-require-execute-multiple-times)